docs/rancher/cloud-provider.md

```diff
@@ -47,41 +47,92 @@ When spinning up an RKE cluster using the Harvester node driver, you can perform
 1. Select `Harvester(Out-of-tree)` option.
 
     ![](/img/v1.2/rancher/rke-cloud-provider.png)
-  
+
 2. Install `Harvester Cloud Provider` from the Rancher marketplace.
 
     ![](/img/v1.2/rancher/install-harvester-cloud-provider.png)
 
-  
+
 ### Deploying to the RKE2 Cluster with Harvester Node Driver
 
 When spinning up an RKE2 cluster using the Harvester node driver, select the `Harvester` cloud provider. The node driver will then help deploy both the CSI driver and CCM automatically.
 
   ![](/img/v1.2/rancher/rke2-cloud-provider.png)
 
-### Deploying to the RKE2 custom cluster (experimental)
+### Deploying to the RKE2 custom cluster with Harvester Cloud Provider (experimental)
 
 ![](/img/v1.2/rancher/custom.png)
 1. Use `generate_addon.sh` to generate a cloud-config and place it into the directory `/etc/kubernetes/cloud-config` on every custom node.
 
-   ```
-   curl -sfL https://raw.githubusercontent.com/harvester/cloud-provider-harvester/master/deploy/generate_addon.sh | bash -s <serviceaccount name> <namespace> 
-   ```
+    ```
+    curl -sfL https://raw.githubusercontent.com/harvester/cloud-provider-harvester/master/deploy/generate_addon.sh | bash -s <serviceaccount name> <namespace>
+    ```
+
+    The output will look as follows:
```

Comment:
```suggestion
  :::note
  The script depends on `kubectl` and `jq` when operating the Harvester cluster, and functions only when given access to the `Harvester Cluster` kubeconfig file. You can find the kubeconfig file in one of the Harvester management nodes (path: `/etc/rancher/rke2/rke2.yaml`).

  You must specify the namespace in which the guest cluster will be created.
  :::

  Example of output:
```

Reply:
I moved the note here because the information must be closer to the main instruction and the actual command.

Reply:
Hi @jillian-maroket, I move the note to close to the command. Please take a look again. Thank you.